### ISSUE TITLE
Can't pass graph model test

### ISSUE THREAD

**Issue by Chasego** [fchollet/keras#930]
I'm new to keras. Recently, I ran an automatic test, ```keras\tests\auto\test_graph_model.py```. 

It fails only because of the function ```test_2o_1i_weights(self)``` inside this file. Specifically, it only can't pass the test ```test a non-sequential graph with 1 input and 2 outputs``` because of the Assertion (loss < 4.0). Since all tests could pass after I commented on the line ```assert (loss < 4.)```

![image](https://cloud.githubusercontent.com/assets/1727284/10870630/767c1bd8-809d-11e5-8cae-be07e319607b.png)

============================================================================
The result for ```test a non-sequential graph with 1 input and 2 outputs```  is shown below.

![test_graph_model_fail](https://cloud.githubusercontent.com/assets/1727284/10870653/0b2a0d4e-809e-11e5-99e1-dde50f9d2f44.PNG)

I appreciate any ideas about it. Some setting are as follows.

1. Keras version: 0.2.0

2. OS: Windows 8.1 (I have to dual-boot on this PC with the pre-installed Windows later. At least now, I can't use Linux on VM lack of GPU visualization.)

3.  Theano version
![image](https://cloud.githubusercontent.com/assets/1727284/10870695/c552364c-809e-11e5-97e9-a1a9d06513ce.png)

**Comment by fchollet** [comment]
These types of test might sometimes fail on untested configurations due to different RNG behaviors. Don't read too much into it.

**Comment by Chasego** [comment]
RNG is random number generation?

**Comment by fchollet** [comment]
Yes.
